### PR TITLE
Add team component/Conditional social urls

### PR DESCRIPTION
**Pull request by WellingGuzman**
On `src/components/Team/SocialIcon.js` I made some conditionals to make sure it matches the icons it used, we can also make a object to map the "keys" to the social icons

### CODE REVIEW

**Comment by RichardLitt** [comment]
This looks good to me! Awesome work.

File: src/components/Team/Person.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import SocialIcons from './SocialIcons'
+
+const Person = (props) => {
+  return (
+    <div className="col-lg-3">
+      <div className="webdeveloper text-center">
+        <img src={props.avatar} className="img-fluid rounded-circle avatar" alt={props.name} />
+        <h4>{props.name}</h4>
+        <p className="text-muted">{props.jobTitle}<br />{props.jobSubTitle}</p>
+        <p className="bio">Full stack developer, nomad, poet, sailor, linguist, writer, and runner. Always making new projects, always iterating on the old ones.</p>
```
Comment (WellingGuzman) [inline src/components/Team/Person.js]: ugh, I forgot to get this from the actual json.

**Comment by WellingGuzman** [comment]
Thanks @RichardLitt!

**Comment by VernL** [comment]
Nice one!

**Pull request merged**